auctions/README.md

```diff
@@ -0,0 +1,9 @@
+# Overview
```

Comment:
I suggest looking at the other README files and adjusting this one to contain a higher level structure / scope / content of the auctions as this only has a few lines / bullet points. It's too barebones.